tests/constraint/test_SO2UniformSampler.cpp

```diff
@@ -26,7 +26,7 @@ class SO2UniformSamplerTests : public ::testing::Test
     mRng = make_unique<RNGWrapper<std::default_random_engine>>(0);
 
     mTargets.clear();
-    for (int i = 0; i < NUM_TARGETS; ++i)
+    for (auto i = 0u; i < NUM_TARGETS; ++i)
```

Comment:
Nit: Could we just make this a `size_t`? I'm generally a fan of [almost always `auto`](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/), but it's a bit funny that we changed a bunch of other loop variables to `size_` in this pull request. 😉